packages/components/tests/dummy/app/templates/components/button-set.hbs

```diff
@@ -0,0 +1,77 @@
+{{page-title "ButtonSet Component"}}
+
+<h2 class="dummy-h2">ButtonSet</h2>
+
+<section>
+  <h3 class="dummy-h3" id="overview"><a href="#overview" class="dummy-link-section">§</a> Overview</h3>
+
+  <p class="dummy-paragraph">Provides consistent layout and spacing of buttons.</p>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="component-api"><a href="#component-api" class="dummy-link-section">§</a> Component API</h3>
+  <p class="dummy-paragraph" id="component-api-button-set">
+    Here is the API for the component:
+  </p>
+  <dl class="dummy-component-props" aria-labelledby="component-api-button-set">
+    <dt>...attributes</dt>
+    <dd>
+      <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+    </dd>
+  </dl>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="how-to-use"><a href="#how-to-use" class="dummy-link-section">§</a> How to use</h3>
+
+  {{! prettier-ignore-start }}
+  <CodeBlock
+    @language="markup"
+    @code='
+      <Hds::ButtonSet>
+        <Hds::Button @text="Button 1" />
+        <Hds::Button @text="Button 2" />
+        <Hds::Button @text="Button 3" />
+      </Hds::ButtonSet>
+    '
+  />
+  {{! prettier-ignore-end }}
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="design-guidelines">
+    <a href="#design-guidelines" class="dummy-link-section">§</a>
+    Design guidelines
+  </h3>
+
+  <div class="dummy-design-guidelines">
+    <p class="dummy-paragraph">
+      <a
+        href="https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?node-id=2%3A9"
+        target="_blank"
+        rel="noopener noreferrer"
+      >Figma UI Kit</a>
+    </p>
+    {{!
+    <br />
+    <img class="dummy-figma-docs" src="/assets/images/button-set-design-usage.png" alt="" role="none" />
+    }}
+  </div>
+</section>
+
+{{! <section>
+  <h3 class="dummy-h3" id="accessibility"><a href="#accessibility" class="dummy-link-section">§</a> Accessibility</h3>
+  <p class="dummy-paragraph">
+    This component has been designed and implemented with accessibility in mind. When used as recommended, there should not be any WCAG conformance issues with this component.
+  </p>
+</section> }}
+
+<section data-test-percy>
+  <h3 class="dummy-h3" id="showcase"><a href="#showcase" class="dummy-link-section">§</a> Showcase</h3>
+
+  <Hds::ButtonSet>
+    <Hds::Button @text="Button 1" />
+    <Hds::Button @text="Button 2" />
+    <Hds::Button @text="Button 3" />
+  </Hds::ButtonSet>
+</section>
```

Comment:
Is this an ok example? Are there other more real world related examples you think would be helpful to add?

Reply:
Yes, having a more realistic use case would be better  (maybe just a primary and a secondary button?)